jaxtyping/array_types.py

```diff
@@ -88,9 +88,7 @@ def __init__(self, expr, broadcastable):
 
 
 def _check_dims(
-    cls_dims: List[_AbstractDim],
-    obj_shape: Tuple[int],
-    single_memo: Dict[str, int],
+    cls_dims: List[_AbstractDim], obj_shape: Tuple[int], single_memo: Dict[str, int],
```

Comment:
Can this unrelated change be reverted?

Reply:
Regarding CLA - thought I have signed that already. Otherwise feel free to do it yourself :)